todo_app/templates/index.html

```diff
@@ -10,10 +10,36 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col">
       <h2>Completed Items</h2>
       <ul class="list-group mb-4">
-        {% for item in complete_items %}
+        {% for item in view_model.complete_items %}
```

Comment:
My problem not yours, but I think this may be the first time I've ever seen completed items before to-do items, and it left me confused for a good minute!